CHANGELOG.rst

```diff
@@ -2,6 +2,353 @@
+external systems to retrieve pricing information!
+
+**🛑 Limiting the amount of submissions**
+
+You can now specify a maximum number of submissions for a given form, useful for limited availability/capacity
+situations, such as raffles or sign-ups to events. Related to that, we expanded the statistics to allow exporting
+the successfully registered submissions.
+
+**🤖 Automatic technical configuration**
+
+We're shipping some tooling for infrastructure teams that deploy Open Forms - this makes it possible to
+provision some configuration aspects that previously had to be done in the admin interface via point-and-click.
+
+We're still expanding on the covered configuration aspects, so stay tuned for more!
+
+**🚸 User Experience improvements**
+
+We have made a ton of user experience improvements in registration and prefill plugin configurations! No
+more copying of URLs from other systems - instead, you select the relevant option in a dropdown.
+Dropdowns that support a search field to wade through those tens or hundreds of available case types!
+
+And, wherever you need to choose a form variable, you now have the options grouped by type of variable
+*and* the context of where this variable occurs, topped of with a search field.
+
+Detailed changes
+----------------
+
+**New features**
+
+* [#4969] Improved the UX of the form designer:
+
+    - The base form configuration tab now groups related fields and collapses them to declutter the UI.
+    - Moved the introduction page configuration to clarify the difference with the introduction text fields.
+* Registration plugins:
+
+    * [:backend:`4686`] All the registration plugin configuration options are now consistently managed in a 
+      modal with better UX.
+
+    * Email:
+
+        * [:backend:`4650`] The email registration plugin now allows setting the recipient using form variables.
+    * Objects API:
+
+        * [:backend:`4978`] Made v2 config the default one.
+        * Updated technical configuration documentation for Objects API.
+        * [:backend:`4398`] The object ownership is now checked, when a user is authenticated, during prefill
+          and pre-registration.
+        * [:backend:`4418`] You can now map individual parts of the addressNL component.
+        * [:backend:`4396`, :backend:`4693`, :backend:`4608`, :backend:`4859`] You can now configure a variable
+          to be prefilled from the Objects API (a.k.a. "product prefill"):
+
+          - It's possible to assign individual properties from the object type to particular form variables.
+          - To avoid duplicating configuration, you can copy the configuration from a configured registration
+            backend. The button is shown only when there are available backends configured.
+          - Cleaned up API of VariablesMapping component.
+          - Refactored VariablesMapping component.
+          - Refactored ObjectTypeVersionSelect.
```

Comment:
Belongs in the prefill group, not registration :)

```suggestion
        * [:backend:`4396`, :backend:`4693`, :backend:`4608`, :backend:`4859`] You can now configure a variable
          to be prefilled from the Objects API (a.k.a. "product prefill"):

          - It's possible to assign individual properties from the object type to particular form variables.
          - To avoid duplicating configuration, you can copy the configuration from a configured registration
            backend.
```